src/constants/WalletAndCurrencyConstants.js

```diff
@@ -146,7 +148,10 @@ export const SPECIAL_CURRENCY_INFO: SpecialCurrencyInfo = {
       modalMessage: s.strings.request_xlm_minimum_notification_body
     },
     displayBuyCrypto: false,
-    isImportKeySupported: true
+    isImportKeySupported: {
+      privateKeyLabel: 'create_wallet_import_input_prompt',
```

Comment:
This should be the string itself ie. `s.string.create_wallet_import_input_prompt`